sources/packages/backend/apps/api/src/services/application-restriction-bypass/application-restriction-bypass.service.ts

```diff
@@ -0,0 +1,49 @@
+import { Injectable } from "@nestjs/common";
+import {
+  ApplicationRestrictionBypass,
+  ApplicationStatus,
+  RecordDataModelService,
+} from "@sims/sims-db";
+import { ApplicationRestrictionBypassSummaryAPIOutDTO } from "apps/api/src/route-controllers/application-restriction-bypass/models/application-restriction-bypass.dto";
+import { DataSource } from "typeorm";
+
+/**
+ * Service layer for application restriction bypasses.
+ */
+@Injectable()
+export class ApplicationRestrictionBypassService extends RecordDataModelService<ApplicationRestrictionBypass> {
+  constructor(dataSource: DataSource) {
+    super(dataSource.getRepository(ApplicationRestrictionBypass));
+  }
+
+  /**
+   * Returns all restriction bypasses for a given application.
+   * @param applicationId id of the application to retrieve restriction bypasses.
+   * @returns application restriction bypasses.
+   */
+  async getApplicationRestrictionBypasses(
+    applicationId: number,
+  ): Promise<ApplicationRestrictionBypassSummaryAPIOutDTO[]> {
+    return this.repo
+      .createQueryBuilder("applicationRestrictionBypass")
```

Comment:
Please use the typeorm find unless there is a reason to not use it. Here is a sample for the find [Sample Code](https://github.com/bcgov/SIMS/blob/f1f0e615c3fc8491b168598b9f906dbc22265918/sources/packages/backend/apps/api/src/services/institution-location/institution-location.service.ts#L229)

Reply:
Updated.